bin/lib/book.ml

```diff
@@ -138,10 +143,11 @@ let next_chapter_footer next_chapter : Html.item option =
     front page and only chapter pages contain links to a next chapter,
     so these are additional arguments. *)
 let main_template ?(next_chapter_footer=None)
+    ?page_title
```

Comment:
I'm pretty sure this should be chapter_title, not page_title.  page_title will be computed from this.